test/api/add.spec.js

```diff
@@ -21,13 +21,13 @@ if (isNode) {
 }
 
 describe('.add', function () {
+  this.timeout(60000)
+
   it('add file', function (done) {
```

Comment:
Please move these `function`s to `=>` functions as the only reason I kept the inconsistency was the need for calling `this.timeout`